backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.crm.model.db;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity(name = "car_entity")
+public class CarEntity {
+
+    @Id
+    private int id;
```

Comment:
Shouldn't be Long id? JpaRepository requires Long for findById() method

Reply:
done